### PR TITLE
Editorial

### PR DESCRIPTION
**Pull request by cjpatton**

### CODE REVIEW

File: draft-ietf-ppm-dap.md [hunk 1]
```diff
@@ -1018,34 +1018,34 @@ Otherwise, `out` is the pair `(new_state, prep_msg)`, where `new_state` is its u
...
-The leader then sends each PrepareStep to the helper in an AggregateContinueReq message,
+The leader then sends each PrepareStep to the helper in an AggregateContReq message,
 structured as follows:
 
 ~~~
 struct {
   AggregationJobID job_id;
   PrepareStep prepare_shares<1..2^16-1>;
-} AggregateContinueReq;
+} AggregateContReq;
```
[the remark this answers is not in the document]
Comment (cjpatton) [reply draft-ietf-ppm-dap.md:1028]: Then I would suggest changing "AggregateInit" to "AggregateInitialize" for consistency. (My preference would be shorter names for both, but my main concern is being consistent.)

[the remark this answers is not in the document]
Comment (chris-wood) [reply draft-ietf-ppm-dap.md:1028]: Making them consistent seems good 👍

**Review by chris-wood — COMMENTED** [review]
The s/PPM/DAP/g changes look good.

File: draft-ietf-ppm-dap.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (cjpatton) [reply draft-ietf-ppm-dap.md]: Done

**Pull request merged**